app/lib/frontend/templates/layout.dart

```diff
@@ -50,7 +51,9 @@ String renderLayoutPage(
       : serializeSearchOrder(searchQuery.order);
   final platformDict = getPlatformDict(platform);
   final isRoot = type == PageType.landing && platform == null;
+  final bodyClass = requestContext.isExperimental ? 'experimental' : '';
   final values = {
+    'body_class': bodyClass,
```

Comment:
nit, but why not just `'body_class': requestContext.isExperimental ? 'experimental' : ''`